cmake/Hunter/config.cmake

```diff
@@ -4,12 +4,6 @@ hunter_config(
     CMAKE_ARGS CMAKE_POSITION_INDEPENDENT_CODE=ON
 )
 
-hunter_config(
```

Comment:
Why is it deleted?

Reply:
hunter already has a GTest version, which is used in libp2p. I don't think we should use different version in kagome.

Reply:
ok, put back that version